pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -37,6 +37,34 @@ type MPIJobList struct {
 	Items           []MPIJob `json:"items"`
 }
 
+// RunPolicy encapsulates various runtime policies of the distributed training
+// job, for example how to clean up resources and how long the job can stay
+// active.
+type RunPolicy struct {
+	// CleanPodPolicy defines the policy to kill pods after the job completes.
+	// Default to Running.
+	CleanPodPolicy *common.CleanPodPolicy `json:"cleanPodPolicy,omitempty"`
```

Comment:
Sure, it wasn't clear to me from the so far discussions so did it with minimal changes required to place the suspend field, but let me update.

Reply:
It might be better to copy all types, and members to this repo since moving only part of those might make double management.

cc: @alculquicondor